aggregator/src/aggregator.rs

```diff
@@ -2583,7 +2591,7 @@ impl VdafOps {
 
     /// Handle GET requests to a collection job URI obtained from the leader's `/collect` endpoint.
     /// The return value is an encoded `CollectResp<Q>`.
-    /// https://www.ietf.org/archive/id/draft-ietf-ppm-dap-02.html#section-4.5.1
+    /// <https://www.ietf.org/archive/id/draft-ietf-ppm-dap-02.html#section-4.5.1>
```

Comment:
The mention of `/collect` is obsolete, as is the reference to draft 2.

Reply:
Good catch, I'll do another pass for this.